content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -493,4 +492,4 @@ the `.spec.replicas` field automatically.
   Read the {{< api-reference page="workload-resources/stateful-set-v1" >}}
   object definition to understand the API for stateful sets.
 * Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
-  you can use it to manage application availability during disruptions.
\ No newline at end of file
+  you can use it to manage application availability during disruptions.
```

Comment:
what are we updating in this line? It seems to me that nothing has changed here

Reply:
i have not updated that but i don't know how this showing here

Reply:
It seems like you have done it by mistake . Not an issue . Its ok.